### PR TITLE
[keyboard] added bear_face PCB

### PR DESCRIPTION
**Pull request by chemicalwill**
Added bear_face PCB

* ATMega32u4-based replacement PCB for the Vortex Race 3

* Default keymap emulates stock board keymap
    - included persistent base layer toggles for QWERTY, COLEMAK, and DVORAK layouts
    - Caps Lock set to LT(_FN1, KC_CAPS), Fn key set to MO(_FN1)
    - "Pn" key set to KC_NO by default

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Adds support for bear_face keyboard

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bear_face/bear_face.c [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "bear_face.h"
+
+void keyboard_pre_init_kb(void) {
+    //Calls the keyboard pre init code
+    //Sets LED pin as output
+setPinOutput(F7);
+}
```
Comment (zvecr) [inline keyboards/bear_face/bear_face.c]:
```suggestion
void keyboard_pre_init_kb(void) {
    //Sets LED pin as output
    setPinOutput(F7);

    keyboard_pre_init_user();
}
```

File: keyboards/bear_face/info.json [hunk 2]
```diff
@@ -0,0 +1,824 @@
+[
```
Comment (zvecr) [inline keyboards/bear_face/info.json]: This file has the wrong format <https://docs.qmk.fm/#/reference_configurator_support>. It looks like KLE so <https://qmk.fm/converter/> might help.

File: keyboards/bear_face/rules.mk [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality on F7 by default
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI controls
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+
+# Layouts used in keymap
+LAYOUTS = 83_ansi
```
Comment (zvecr) [inline keyboards/bear_face/rules.mk]: This is for <https://docs.qmk.fm/#/feature_layouts>, to which 83_ansi isnt currently one of the supported

```suggestion
AUDIO_ENABLE = no           # Audio output on port C6
```

Comment (chemicalwill) [reply keyboards/bear_face/rules.mk]: Makes sense - I saw that somewhere else and was emulating it.

File: keyboards/bear_face/bear_face.c (hunk 1 above)
Comment (chemicalwill) [reply keyboards/bear_face/bear_face.c]: fixed

File: keyboards/bear_face/readme.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# bear_face
+
+![bear_face](https://imgur.com/a/11fNXvS)
+===
```
Comment (fauxpark) [inline keyboards/bear_face/readme.md]:
```suggestion
![bear_face](https://i.imgur.com/nQIyP6w.png)
```

File: keyboards/bear_face/rules.mk [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE = yes       # Mouse keys(+4700)
+EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
+CONSOLE_ENABLE = no         # Console for debug(+400)
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality on F7 by default
```
Comment (fauxpark) [inline keyboards/bear_face/rules.mk]:
```suggestion
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
```

File: keyboards/bear_face/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,98 @@
+#include QMK_KEYBOARD_H
+
+#define _QWER 0
+#define _COLE 1
+#define _DVOR 2
+#define _FN1 3
```
Comment (drashna) [inline keyboards/bear_face/keymaps/default/keymap.c]: Especially for the default keymap, I'd rather this be an enum, so it's easier to manage: 

```suggestion
enum layers {
    _QWER,
    _COLE,
    _DVOR,
    _FN1,
};
```

File: keyboards/bear_face/info.json [hunk 7]
```diff
@@ -0,0 +1,12 @@
+{
+  "keyboard_name": "bear_face", 
+  "url": "https://github.com/chemicalwill/bear_face_pcb", 
+  "maintainer": "chemicalwill", 
+  "width": 16, 
+  "height": 28, 
+  "layouts": {
+      "LAYOUT": {
```
Comment (zvecr) [inline keyboards/bear_face/info.json]:
```suggestion
      "LAYOUT_83_ansi": {
```

Comment (chemicalwill) [reply keyboards/bear_face/info.json]: So I had removed `LAYOUTS = 83_ansi` from `rules.mk` but I'm guessing this serves a different purpose? I don't know a ton about what's going on under the hood here but would love to learn

**Comment by chemicalwill** [comment]
@zvecr I had removed LAYOUTS = 83_ansi from `rules.mk` but I'm guessing there in `info.json` it serves a different purpose? I don't know a ton about what's going on under the hood here but would love to learn

**Comment by chemicalwill** [comment]
I would like to test compile and change the license and readme to use my handle name before merging if that is allowed

**Comment by chemicalwill** [comment]
Checked compile, everything seems to work great

**Pull request merged**